src/Typeahead.react.js

```diff
@@ -101,8 +102,8 @@ class Typeahead extends React.Component {
         <Overlay
           {...overlayProps}
           container={bodyContainer ? document.body : this}
-          referenceElement={this._inputContainer}
-          show={isMenuShown}>
+          referenceElement={this._inputContainer ? this._inputContainer : alwaysOpen}
```

Comment:
This is needed because upon first render, the referenceElement prop comes through to the `<Overlay>` as `undefined` due to a race condition with the ref instantiation above. On any further interaction to or within the plugin, the Overlay would pop open and respect the new property. But I found that even though it is `undefined` on first render, it does have access to the ref and can render, so this is a little shim to get past that nuance, and achieves a proper initial render state.